node/device/tun/wintun/wintun.go

```diff
@@ -138,10 +139,12 @@ func (wintun *Adapter) Close() (err error) {
 
 // Uninstall removes the driver from the system if no drivers are currently in use.
 func Uninstall() (err error) {
-	r1, _, e1 := syscall.Syscall(procWintunDeleteDriver.Addr(), 0, 0, 0, 0)
```

Comment:
I don't think we need to ignore error here and it's better to ignore error at `_ = wintun.Uninstall()`.

Reply:
Makes sense